.travis.yml

```diff
@@ -1,6 +1,5 @@
 ---
 language: go
 go:
-  - 1.3
-  - 1.4
+  - 1.7
```

Comment:
Go 1.7 [moves](https://golang.org/doc/go1.7#context) the `golang.org/x/net/context` package into the standard library as `context`. Because of that, `import "context"` in the `safe_prime_generator.go` does not compile on Go versions prior to 1.7. 

There is no easy way to support older versions without sacrificing quality. Since Go 1.7 was released two years ago, I think that supporting 1.7+ is an acceptable move.

Reply:
Totally okay with me.